### PR TITLE
Convert ISS notebook to source data from the cloud.

### PR DESCRIPTION
**Pull request by ttung**
The stack is read fairly normally.  The codebook is currently sourced through a side channel, so I need to think of a better way to handle that. :)

Depends on #146 

You can look at s3://czi.starfish.data.public/ISS/ to see how to lay out your files (there's really nothing special, since #146 grants us the ability to resolve like a web browser)

### CODE REVIEW

File: notebooks/ISS_Pipeline_-_Breast_-_1_FOV.ipynb [hunk 1]
```diff
@@ -1157,17 +1157,17 @@
        "VEGF      16\n",
        "MKI67     15\n",
        "CCND1     11\n",
-       "GUS        8\n",
        "STK15      8\n",
-       "AIB1       7\n",
+       "GUS        8\n",
        "ETV4       7\n",
```
[the remark this answers is not in the document]
Comment (ambrosejcarr) [reply notebooks/ISS_Pipeline_-_Breast_-_1_FOV.ipynb]: It looks like it's the same by value, but the order has changed. Great question about why this happens, though!

File: notebooks/ISS_Pipeline_-_Breast_-_1_FOV.py [hunk 2]
```diff
@@ -85,7 +85,7 @@
 # EPY: END markdown
 
 # EPY: START code
-codebook = pd.read_csv('ISS/codebook.csv', dtype={'barcode': object})
+codebook = pd.read_csv('http://czi.starfish.data.public.s3-website-us-east-1.amazonaws.com/ISS/codebook.csv', dtype={'barcode': object})
```
Comment (ambrosejcarr) [inline notebooks/ISS_Pipeline_-_Breast_-_1_FOV.py]: This is cool. I didn't realize that pandas could resolve http files.

File: notebooks/ISS_Pipeline_-_Breast_-_1_FOV.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (ttung) [reply notebooks/ISS_Pipeline_-_Breast_-_1_FOV.ipynb]: Hash tables!

[the remark this answers is not in the document]
Comment (ttung) [reply notebooks/ISS_Pipeline_-_Breast_-_1_FOV.ipynb]: #153

**Pull request merged**